generators/app/resources/plugins.js

```diff
@@ -27,58 +25,63 @@ AWS.config.update(_.merge({
     agent: new https.Agent({
```

Comment:
i think this protocol stuff is no longer needed with latest aws sdk and node. At least i know we're not doing it anymore in our most recent projects and everything is working fine